fusee/fusee-secondary/src/nxboot.c

```diff
@@ -607,11 +609,21 @@ uint32_t nxboot_main(void) {
         }
 
         if (tsec_fw_size == 0x3000) {
-            sept_secondary_enc = sept_secondary_00_enc;
-            sept_secondary_enc_size = sept_secondary_00_enc_size;
+            if (!fuse_get_retail_type()) {
```

Comment:
invert this if. The retail case should be the if, and the dev case should be the else.